migrations/20240916110123_create_items_table.js

```diff
@@ -0,0 +1,40 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('items', function (table) {
+    table.increments('id').primary();
+    table
+      .integer('owner_id')
+      .unsigned()
+      .notNullable()
+      .references('id')
+      .inTable('users')
+      .onDelete('CASCADE');
```

Comment:
```suggestion
```

We don't want to delete all items if a user deleted. Let's assume a user has 100 files on S3. If we delete all item records on database when you delete the user, 100 files would be stayed on S3. We don't wanna have garbage data on S3. No auto-deletion.